src/main/groovy/io/seqera/wave/service/metric/impl/MetricsServiceImpl.groovy

```diff
@@ -127,4 +138,13 @@ class MetricsServiceImpl implements MetricsService {
         return null
     }
 
+    protected static String extractOrgFromKey(String key) {
+        Matcher matcher = orgDateKeyPattern.matcher(key)
+        if (matcher.matches()) {
+            return matcher.group(2)
+        } else {
+            return "unknown"
+        }
```

Comment:
prefer conditional expression to make it more concise/readable

Reply:
done

Reply:
Nice